make_graph.py

```diff
@@ -1,61 +1,222 @@
+for bucket in all_s3_buckets:
     s3_cluster.node(bucket.bucket_name, shape="cylinder")
-
 dot.subgraph(s3_cluster)
 
-queries_subgraph = Digraph(name="cluster_queries")
-queries_subgraph.attr(label="Named Queries")
-queries_subgraph.attr(style="filled", color="lightgrey", fontsize="16")
-
-database_clusters = {}
-for database in DATABASES:
-    database_clusters[database.database_name] = Digraph(
-        name=f"cluster_{database.database_name}"
-    )
-    database_clusters[database.database_name].attr(
-        label=f"Database: {database.database_name}"
-    )
-    database_clusters[database.database_name].attr(
-        style="filled", color="lightgrey", fontsize="16"
-    )
-
-# Add nodes and edges based on dependencies
-for table in TABLES:
-    database_clusters[table.database.database_name].node(
-        table.table_name, shape="component"
-    )
-    if table.depends_on:
-        for dep in table.depends_on:
-            dot.edge(table.table_name, dep.table_name)
-    dot.edge(table.table_name, table.bucket.bucket_name, label=table.s3_prefix)
-
-    if table.populated_with:
-        queries_subgraph.node(table.populated_with.name)
-        dot.edge(table.table_name, table.populated_with.name)
-        if "from_table" in table.populated_with.context:
-            dot.edge(
-                table.populated_with.name,
-                table.populated_with.context["from_table"],
-            )
-
-for database_cluster in database_clusters.values():
-    dot.subgraph(database_cluster)
-
+#
+# queries_subgraph = Digraph(name="cluster_queries")
+# queries_subgraph.attr(label="Named Queries")
+# queries_subgraph.attr(style="filled", color="lightgrey", fontsize="16")
+#
+# database_clusters = {}
+# for database in DATABASES:
+#     database_clusters[database.database_name] = Digraph(
+#         name=f"cluster_{database.database_name}"
+#     )
+#     database_clusters[database.database_name].attr(
+#         label=f"Database: {database.database_name}"
+#     )
+#     database_clusters[database.database_name].attr(
+#         style="filled", color="lightgrey", fontsize="16"
+#     )
+#
+# # Add nodes and edges based on dependencies
+# for table in TABLES:
+#     database_clusters[table.database.database_name].node(
+#         table.table_name, shape="component"
+#     )
+#     if table.depends_on:
+#         for dep in table.depends_on:
+#             dot.edge(table.table_name, dep.table_name)
+#     dot.edge(table.table_name, table.bucket.bucket_name, label=table.s3_prefix)
+#
+#     if table.populated_with:
+#         queries_subgraph.node(table.populated_with.name)
+#         dot.edge(table.table_name, table.populated_with.name)
+#         if "from_table" in table.populated_with.context:
+#             dot.edge(
+#                 table.populated_with.name,
+#                 table.populated_with.context["from_table"],
+#             )
+#
+# for database_cluster in database_clusters.values():
+#     dot.subgraph(database_cluster)
+#
```

Comment:
does this big block of commented code need to be here?

Reply:
Nope!